main.nf

```diff
@@ -13,9 +13,9 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-include { NF-CHAI  } from './workflows/nf-chai'
-include { PIPELINE_INITIALISATION } from './subworkflows/local/utils_nfcore_nf-chai_pipeline'
-include { PIPELINE_COMPLETION     } from './subworkflows/local/utils_nfcore_nf-chai_pipeline'
+include { NF_CHAI  } from './workflows/nf_chai'
```

Comment:
No Harshil alignment

Reply:
I am going to fix that in the next PR!! Mwahahaahahaha! This one was just to delete excessive files and get the tests running again.